osi_hostvehicledata.proto

```diff
@@ -2,31 +2,356 @@ syntax = "proto2";
+
+            // Contact point of the mid of the tire tread with the world.
+            // Absolute coordinates (x, y, z).
+            //
+            // Unit: m
+            //
+            //optional Vector3d contact_point = 4;
+
+            // Contains the rotation rate of the wheel per second in radians.
+            // Based on the processed output of the hall sensor at the wheel.
+            //
+            // Unit: rad/s
+            //
+            optional double rotation_rate = 5;
+
+            // Contains the longitudinal slip of the tire.
+            // \par References:
+            // - https://www.kfz-tech.de/Biblio/Formelsammlung/Schlupf.htm
+            //
+            // Unit: %
+            //
+            optional double slip = 6;
+        }
+    }
+
+    //
+    // \brief This message contains all the information the vehicle knows about its kinematic states.
+    //
+    message VehiclePositionAndKinematics
+    {
+        //
+        // \brief Current kinematic data based on the output of the Intertial Measurement Unit (IMU).
+        //
+        message CartesianInformation
+        {
+            // Current kinematic data based on the output of the Intertial Measurement Unit (IMU).
+            //
+            // \note Note that dimension and base_polygon need not be set.
+            //
+            optional BaseMoving cartesian_data = 1;
+
+            // Current kinematic data error based on the output of the Intertial Measurement Unit (IMU).
+            //
+            // \note Note that dimension and base_polygon need not be set.
+            //
+            optional BaseMoving cartesian_data_rmse = 2;
+        }
+
+        //
+        // \brief Current estimated location based on the output of the GPS-Unit.
+        //
+        message GeoreferencedInformation
+        {
+            // Longitude in decimal degrees regarding WGS84.
+            //
+            // Unit: Degree
+            // Range: [-180; 180]
+            //
+            optional double longitude = 1;
+
+            // Latitude in decimal degrees regarding WGS84.
+            //
+            // Unit: Degree
+            // Range: [-90; 90]
+            //
+            optional double latitude = 2;
+
+            // Height above sea level regarding EGM96.
+            //
+            // Unit: m
+            // Range: [-300; 10000]
+            //
+            optional double altitude = 3;
+
+            // To be discussed as it is in CartesianInformation. 
+            // Heading in decimal degrees.
+            //
+            // Unit: Degree
+            // Range: [0; 360]
+            //
+            // optional double heading = 4;
+
+            // Accuracy of localization measurement in percentage of the units.
+            //
+            // Unit: %
+            //
+            // optional double localization_accuracy = 5;
+
+            // Number of satellites.
+            //
+            // optional int32 number_of_satellites = 6;
```

Comment:
are these fields supposed to be commented out?

Reply:
From my point of new not. At least having the heading in the message would be beneficial for sensor systems.

Reply:
Actually there are sensor informations in OSI also like lidar and camera. I would suggest to rather add an additional sensor of type GNSS and add this information there. Also if we use a geodetic reference to CartesianInformation like suggested in #486 then GeoreferencedInformation would not be needed anyway, as you could set the reference to you geodetic position and the cartesian position to zero which is basically the same result as only specifying the geodetic position itself.

Reply:
Yes, i also think a _"SensorData GNSS"_ is the most plausible way.